.github/workflows/release.yaml

```diff
@@ -1,14 +1,19 @@
 name: Release GoPowermax
 # Invocable as a reusable workflow
 # Can be manually triggered
-on:  # yamllint disable-line rule:truthy
+on: # yamllint disable-line rule:truthy
```

Comment:
Can this change be reverted to fix lint warning?

Reply:
Updated in recent commit.